lib/xcodeproj/plist.rb

```diff
@@ -1,10 +1,10 @@
+autoload :Nanaimo, 'nanaimo'
+autoload :CFPropertyList, 'cfpropertylist'
```

Comment:
Eh, Id say the overhead is minimal enough that it would be preferable to having problems when you come to loading it later, and there are very few uses of Xcodeproj that don't require some form of reading/writing. But fair enough ¯\_(ツ)_/¯ 👍